### PR TITLE
Routes and Pages Mocked

### PR DESCRIPTION
**Pull request by keawade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # garden-vue
-An implementation of whiteboards/garden in Vue.
+
+> An implementation of whiteboards/garden in Vue.
```
Comment (NathanBland) [inline README.md]: this no longer lives in whiteboards :thinking:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>garden-vue</title>
+    <link rel="stylesheet" href="/static/normalize.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
```
Comment (NathanBland) [inline index.html:7]: Since this is _technically_ your implementation I can't say no... But please, please don't use `jquery` unless there is a very good reason for it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <link rel="stylesheet" href="/static/semantic.min.css">
+    <script src="/static/semantic.min.js"></script>
```
Comment (NathanBland) [inline index.html:9]: Given the build system style of `vue` projects with webpack, please avoid including resources in this way.

Take a look at [these docs](https://simulatedgreg.gitbooks.io/electron-vue/content/docs/using_css_frameworks.html) for a solid example of using a framework.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>garden-vue</title>
+    <link rel="stylesheet" href="/static/normalize.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/static/semantic.min.css">
```
Comment (NathanBland) [inline index.html:8]: Per our previous style conversation/guidelines, _please_ use `mdl-lite`. 

https://getmdl.io/started/index.html see the npm section.

File: src/App.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+</template>
+
+<script>
+import AppHeader from './components/AppHeader'
+
+export default {
+  name: 'app',
+  data () {
+    return {
+      msg: 'Hello, this is your computer speaking.',
```
Comment (NathanBland) [inline src/App.vue]: you can kill this object if you don't have any default data you want to include, or if you don't like it's props you can just make it an empty object.

File: src/auth.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  },
+
+  loggedIn () {
+    return !!localStorage.token
+  },
+
+  onChange () {}
+}
+
+function pretendRequest (email, pass, cb) {
```
Comment (NathanBland) [inline src/auth.js:40]: ahaha nice :+1:

File: src/components/AppHeader.vue [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+<script>
+export default {
+}
+</script>
+
+<style scoped>
+</style>
+
+    <Router>
```
Comment (NathanBland) [inline src/components/AppHeader.vue]: Uh? Why is this here?

File: src/components/AppHeader.vue [hunk 8]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class='app-header'>
+    <div class='ui secondary pointing menu'>
+      <router-link to='/' class='item'>Feed</router-link>
```
Comment (NathanBland) [inline src/components/AppHeader.vue]: I'd like to see another view that is basically like your `following` but as a feed.... Maybe that's what you intended for `following` to be.

File: src/components/Following.vue [hunk 9]
```diff
@@ -0,0 +1,42 @@
+<template>
```
Comment (NathanBland) [inline src/components/Following.vue:1]: I really like this layout :+1:

File: src/components/Profile.vue [hunk 10]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <div class='ui container'>
+      <h1 class='ui header'>Profile</h1>
+    </div>
+    <br />
+    <div class='ui container'>
+      <div class='ui segment thingy'>
+        <img class='ui medium rounded image' src='/static/square-image.png' />
```
Comment (NathanBland) [inline src/components/Profile.vue]: Padding or margin this image, please.

**Review by NathanBland — CHANGES_REQUESTED** [review]
Left some fun comments for you. Overall I like what you have going.

**Comment by NathanBland** [comment]
Also, I'd like you to remove the default `vue` logo :)

File: src/components/AppHeader.vue (hunk 7 above)
Comment (crodeheaver) [reply src/components/AppHeader.vue]: Just in case he needs it for later, of course!

File: index.html (hunk 2 above)
Comment (keawade) [reply index.html:7]: It's necessary for Semantic UI's animations. I'll comment it out for now since those animations aren't being used yet though.

EDIT: Right, mdl... I'll switch it.

File: index.html (hunk 4 above)
Comment (keawade) [reply index.html:8]: Right, forgot about that. Sorry. I'll switch it.

File: src/components/AppHeader.vue (hunk 8 above)
Comment (keawade) [reply src/components/AppHeader.vue]: `Feed` is a chronological feed of all subscribed things. `Following` is a list of people you are following, sorted by latest post by default. This will probably be sortable/filterable to quickly find someone. Each person will have a short blurb for their last post.

File: src/components/AppHeader.vue (hunk 7 above)
Comment (keawade) [reply src/components/AppHeader.vue]: Ahaha, I pasted that temporarily to format for Gitter. Didn't realize I had left it in there.

File: src/components/AppHeader.vue (hunk 8 above)
Comment (NathanBland) [reply src/components/AppHeader.vue]: What if their latest post is of a different content type than `text`? I like where you are going with this, just wanted to see what your thoughts were on this aspect.

Comment (keawade) [reply src/components/AppHeader.vue]: Don't know. I'll iterate on it when I get there.

**Comment by keawade** [comment]
Ready to merge for now.

**Comment by NathanBland** [comment]
I don't feel like fighting the `mdl` battle right now. Have fun
